tools/maxquant/test-data/exp_design_test.txt~

```diff
@@ -0,0 +1,3 @@
+Name	Fraction	Experiment	PTM
```

Comment:
Was this file committed on accident?

Reply:
The file is needed for the doctests in mqparam.py, so not really necessary, but maybe helpful.

Reply:
Sorry, I meant this looks like a temporary editor file. The file without the `~` at the end is also committed.